### PR TITLE
podmanv2 container subcommands

### PR DESCRIPTION
**Pull request by baude**
add container subcommands with example text that has proper context.

Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
Needs a rebase

**Comment by baude** [comment]
fixed and ready

File: cmd/podman/containers/attach.go [hunk 1]
```diff
@@ -26,6 +27,16 @@ var (
...
 	}
+
+	containerAttachCommand = &cobra.Command{
+		Use:   attachCommand.Use,
+		Short: attachCommand.Short,
+		Long:  attachCommand.Long,
+		RunE:  attachCommand.RunE,
+		Example: `podman container attach ctrID
+	podman container attach 1234
+	podman container --no-stdin foobar`,
```
Comment (mheon) [inline cmd/podman/containers/attach.go:38]: Missing an `attach`

**Comment by mheon** [comment]
One problem otherwise LGTM

**Pull request merged**